flexget/plugins/input/trakt_emit.py

```diff
@@ -103,6 +101,8 @@ def on_task_input(self, task, config):
                             else:
                                 epn += 1
                         break
+                    else:
+                        eps = 0;
```

Comment:
not sure why this line ends in a semicolon?

Also you could cleanup the above logic 

IE:
```
if config['position'] == 'next':
                             if seas['completed'] >= seas['aired']:
```

to:
```
if config['position'] == 'next' and seas['completed'] >= seas['aired']:
```